c2rust-analyze/tests/common/mod.rs

```diff
@@ -48,6 +48,18 @@ impl Analyze {
                 "c2rust-analyze failed with status {status}:\n> {cmd:?} > {output_path:?} 2>&1\n"
             );
             let output = fs::read_to_string(&output_path).unwrap();
+            let max_len = 80 * 300; // don't print a ton
+            let output = if output.len() > max_len {
+                let len_truncated = output.len() - max_len;
+                let (truncated, rest) = output.split_at(len_truncated);
+                let num_lines_truncated = truncated.split('\n').count();
+                let truncated_msg = format!(
+                    "truncated the first {len_truncated} bytes ({num_lines_truncated} lines)"
+                );
+                format!("[{truncated_msg}] ...\n{rest}\n[note: {truncated_msg}]")
+            } else {
+                output
+            };
```

Comment:
why?

Reply:
When I was testing which things to uncomment, if there were errors from `lighttpd-minimal`, it would print 20 million bytes to the terminal, so I added this code to truncate it.  I could remove this code, but it seemed useful and I had to write it anyways to not overload my terminal while testing things.

Reply:
I think it's best if we remove it

Reply:
Removed for now, stored in [kkysen/analyze-tests-limit-output](https://github.com/immunant/c2rust/tree/kkysen/analyze-tests-limit-output) in case we ever need it.